backend/venue.js

```diff
@@ -35,41 +35,23 @@ var venue = {
         var first = rows[0];
         // convert the sql results into a json tree
         // begins with venue info
-        var res = {
-          id: first.venue_id,
-          title: first.venue_title,
-          description: first.description,
-          ticket_type: first.ticket_type,
-          sections: _.groupBy(rows, function(row) { return row.section_id;})
-        };
+        var res = _.pick(first, ['venue_title', 'description', 'ticket_type']);
 
         // groupBy creates a dictionary (object) containing lists
         // so convert those lists into objects with section info
-        res.sections = _.mapObject(res.sections, function(rows) {
-          first = rows[0]; // rows is a list of seats
-          var section = {
-            id: first.section_id,
-            title: first.section_title,
-            row_name: first.row_name,
-            seat_count: first.seat_count
-          };
+        var sections = _.groupBy(rows, function(row) { return row.section_id;});
```

Comment:
This is best we can have now, but in the future (ES6, and already in TypeScript I've used) there will be syntax of `(row) => row.section.id` to replace this function :)